en/firefox/browsers/compare/safari.ftl

```diff
@@ -0,0 +1,74 @@
+
+# Variables:
+#   $pocket (string) - link to getpocket.com with additional attributes for analytics
+#   $send (string) - link to send.firefox.com with additional attributes for analytics
+compare-safari-also-when-you-sign-up-for = Also, when you sign up for a { -brand-name-firefox } account you get access to unique services like <a { $pocket }>{ -brand-name-pocket }</a> and <a { $send }>{ -brand-name-send }</a> that integrate directly into the browser. The { -brand-name-pocket } for { -brand-name-firefox } button lets you save web pages and videos to { -brand-name-pocket } in just one click, so you can read a clean, distraction-free version whenever and wherever you want — even offline. With { -brand-name-send } you can share large files with end-to-end encryption with  a variety of security controls, such as the ability to set an expiration time for a file link to expire, the number of downloads, and whether to add an optional password for an extra layer of security.
+
+# Variables:
+#   $pocket (string) - link to getpocket.com with additional attributes for analytics
+#   $send (string) - link to send.firefox.com with additional attributes for analytics
+compare-safari-also-when-you-sign-up-for-fallback = Also, when you sign up for a { -brand-name-firefox } account you get access to some unique services like Screenshots, <a { $pocket }>{ -brand-name-pocket }</a> and <a { $send }>{ -brand-name-send }</a> that integrate directly into the browser. Screenshots is a feature built right into the { -brand-name-firefox } browser, allowing you to copy or download any or all part of a web page. When you save the screenshot, you can also choose which folder you want to find it in, instead of cluttering your desktop. The { -brand-name-pocket } for { -brand-name-firefox } button lets you save web pages and videos to { -brand-name-pocket } in just one click, so you can read a clean, distraction-free version whenever and wherever you want — even offline. With { -brand-name-send }, you can share large files with end-to-end encryption and a variety of security controls, such as the ability to set an expiration time for a file link to expire, the number of downloads, and whether to add an optional password for an extra layer of security.
+
+# Variables:
+#   $attrs (string) - link to screenshots.firefox.com with additional attributes for analytics
+compare-safari-screenshots-is-another-popular = <a { $attrs }>Screenshots</a> is another popular feature built right into the Firefox browser, allowing you to easily capture a picture of your screen. When you take a screenshot, you can choose to copy the file or choose which folder you want to find it in, instead of cluttering your desktop.
```

Comment:
1). `Firefox`
2). Screenshot. Let's decide if we want to include it in the brands file.